dashboard-widgets.el

```diff
@@ -125,21 +134,6 @@ See `create-image' and Info node `(elisp)Image Descriptors'."
   :type 'boolean
   :group 'dashboard)
 
-(defcustom dashboard-set-navigator nil
```

Comment:
I've noted navigator is disabled by default,
Should `dashboard-insert-navigator` be deleted from `dashboard-startupify-list`?

Reply:
If so, then 
Should document all the functions that are avalaibles for dashboard-startupify-list?

Reply:
If so, then dashboard-startupify-list must have a docstring about all the functions avalaibles.

Reply:
Yes, can you applied these changes?

Reply:
Sure.